### ISSUE TITLE
Crash on Connecting to Postgres - Laravel

### ISSUE THREAD

**Issue by JohnDotOwl** [dunglas/frankenphp#514]
When the following database connection code is enable , it will crash the whole octane running franken without any error

```
$data = DB::table('test')->get();
```

Command used to run FrankenPHP
```
php artisan octane:start --caddyfile /var/www/test/Caddyfile --server=frankenphp --log-level=debug
```

If i run it with ./frankenphp run , it will return the following and crash
Segmentation fault

I've installed
sudo apt-get install php8.3-pgsql
sudo apt-get install php8.3-pdo

Update : I've tested it on ARM initially, now x86, it's the same issue.

**Comment by withinboredom** [comment]
Possibly also related to #458.

**Comment by mmccall10** [comment]
FWIW I have a similar issue on a mac M2 with laravel and postgresql. It works fine connecting to a local db but when I attempt to connect using a `DATABASE_URL` to something like a neon endpoint I get this error. 

```sh
frankenphp(8451,0x175243000) malloc: double free for ptr 0x13300c200
frankenphp(8451,0x175243000) malloc: *** set a breakpoint in malloc_error_break to debug
```

**Comment by dunglas** [comment]
This should probably be reported to static-php-cli, or to PHP itself. This smells like an upstream bug.

**Comment by withinboredom** [comment]
> I've installed
> sudo apt-get install php8.3-pgsql
> sudo apt-get install php8.3-pdo

I suspect this is the issue since AFAIK, apt doesn't install php-zts libraries.

**Comment by dunglas** [comment]
Debian packages aren't used and it's useless to install them.

Octane installs the static binary we provide, which is built using ZTS and doesn't use local PHP extensions.

**Comment by kkovacs** [comment]
Just a small update and workaround, based on the thread in `static-php-cli`: the bug seems related to a TLS issue, it doesn't surface if `sslmode=disable` is supplied in the connection string (if you can afford that):

```php
<?php
$conn = pg_connect("postgresql://root:xxxxx@127.0.0.1/postgres?sslmode=disable");
```

**Comment by kkovacs** [comment]
Heads up: the bug was fixed upstream in https://github.com/crazywhalecc/static-php-cli/issues/353 in `static-php-cli`, so hopefully in the next `frankenphp` build it should be fixed too (if it will use `static-php-cli` 2.1.1 or above).

**Comment by dunglas** [comment]
Thank you for getting this fixed @kkovacs!
